### ISSUE TITLE
bug: %-formatted strings not reformatted if only substitution is returned from a function call

### ISSUE THREAD

**Issue by neutrinoceros** [ikamensh/flynt#65]
I've seen some lines using "%" formatting that weren't affected by flynt. Here's my best attempt to reproduce the problem in a minimal form:

```python
from flynt.api import fstringify_code_by_line

# this works perfectly
fstringify_code_by_line("""'%s' % (fn(var),)""", multiline=False, len_limit=100)
>> ("f'{fn(var)}'", 1)

# and this doesn't (it should produce the same result as the first case)
fstringify_code_by_line("""'%s' % fn(var)""", multiline=False, len_limit=100)
>> ("'%s' % fn(var)", 0)
```

**Comment by ikamensh** [comment]
True, ast.Call was not whitelisted on right side of % formatting. Fixed in v0.47.1

**Comment by neutrinoceros** [comment]
Thank you very much !
